src/components/Users/Users.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+
+import styles from './Users.module.css';
+
+const Users = (props) => (
+  <div>
+    {
+                props.users.map((user) => (
+                  <div key={user.id}>
```

Comment:
Consider breaking this JSX into small functional pieces in the current functional components.

Reply:
/fxd